Plugin/CppApi/include/CoordinateConversion/CoordinateConversionResults.h

```diff
@@ -35,11 +35,11 @@ namespace Toolkit
 class Result
 {
 public:
- Result(const QString& name, const QString& notation, int type);
- ~Result() = default;
- QString m_name;
- QString m_notation;
- int m_type; // CoordinateConversionOptions::CoordinateType as int
+  Result(const QString& name, const QString& notation, int type);
+  ~Result() = default;
+  QString m_name;
```

Comment:
I realize it's not in the scope of this PR but seems like either these members should be private with getters/setters as apppropriate or we should make this a `struct` and possibly drop the "m_" prefix

Reply:
A fair point. It's essentially a struct with labelled as a class. Not essentially for this PR.

Reply:
PR for that change: https://github.com/Esri/arcgis-runtime-toolkit-qt/pull/317